PWGHF/TableProducer/candidateCreatorXic0Omegac0.cxx

```diff
@@ -965,15 +1012,15 @@ struct HfCandidateCreatorXic0Omegac0 {
       auto cascNdfm = kfOmegaMassConstrained.GetNDF();
       auto cascChi2OverNdfm = kfOmegac0Candidate.chi2MassCasc / cascNdfm;
 
-      // KF topo Chi2
-      kfOmegac0Candidate.chi2TopoV0ToPv = kfV0ToPv.GetChi2();
-      kfOmegac0Candidate.chi2TopoCascToPv = kfOmegaToPv.GetChi2();
-      kfOmegac0Candidate.chi2TopoPiFromOmegacToPv = kfPiFromOmegacToPv.GetChi2();
-      kfOmegac0Candidate.chi2TopoOmegacToPv = kfOmegac0ToPv.GetChi2();
+      // KF topo Chi2 over NDF
+      kfOmegac0Candidate.chi2TopoV0ToPv = kfV0ToPv.GetChi2() / kfV0ToPv.GetNDF();
+      kfOmegac0Candidate.chi2TopoCascToPv = kfOmegaToPv.GetChi2() / kfOmegaToPv.GetNDF();
+      kfOmegac0Candidate.chi2TopoPiFromOmegacToPv = kfPiFromOmegacToPv.GetChi2() / kfPiFromOmegacToPv.GetNDF();
+      kfOmegac0Candidate.chi2TopoOmegacToPv = kfOmegac0ToPv.GetChi2() / kfOmegac0ToPv.GetNDF();
 
-      auto cascBachTopoChi2 = kfBachKaonToOmega.GetChi2();
-      kfOmegac0Candidate.chi2TopoV0ToCasc = kfV0ToCasc.GetChi2();
-      kfOmegac0Candidate.chi2TopoCascToOmegac = kfOmegaToOmegaC.GetChi2();
+      auto cascBachTopoChi2 = kfBachKaonToOmega.GetChi2() / kfBachKaonToOmega.GetNDF();
+      kfOmegac0Candidate.chi2TopoV0ToCasc = kfV0ToCasc.GetChi2() / kfV0ToCasc.GetNDF();
+      kfOmegac0Candidate.chi2TopoCascToOmegac = kfOmegaToOmegaC.GetChi2() / kfOmegaToOmegaC.GetNDF();
 
```

Comment:
since now you normalize everything by NDF, what about writing it also in the variables name, e.g. `kfOmegac0Candidate.chi2NdfTopoV0ToPv`?